features/facebookImport/src/static/manifest.json

```diff
@@ -6,12 +6,16 @@
     "primaryColor": "#0f1938",
     "links": {
         "data-download": "https://www.facebook.com/dyi",
-        "support-email": "mailto:polypod-feedback@polypoly.coop"
+        "support-email": "mailto:polypod-feedback@polypoly.coop",
+        "example-data-download": "https://polypoly.coop/en-de/polypod/example-facebook-data"
```

Comment:
Totally unrelated to the PR, but why do we use `en-de` as part of the URL? Shouldn't that be `en-uk` or something like that?

Reply:
I forgot, but there was some weird reason :D